components/cam/src/physics/cam/shoc.F90

```diff
@@ -2735,6 +2737,31 @@ subroutine shoc_assumed_pdf_compute_buoyancy_flux(&
 
 end subroutine shoc_assumed_pdf_compute_buoyancy_flux
 
+! Calls the error function. If use_cxx=true, then we call std::erf()
+subroutine error_function(input, output)
```

Comment:
You could make this a function instead of a subroutine, which would make its use a bit more similar to before (i.e., allow to do `c=error_function(blah)`).

Also, along the line of other bfb stuff, why not calling it simply bfb_erf?